docs/user/ppl/cmd/search.rst

```diff
@@ -62,3 +62,274 @@ PPL query::
+    +----------------+-----------+----------------------+---------+--------+--------+----------+-------+-----+----------------------+----------+
+
+Character Escaping and Phrase Search
+=====================================
+
+Phrase Search with Double Quotes
+---------------------------------
+Double quotes in search values indicate phrase search:
+
+* ``search address="Bristol Street" source=accounts`` 
+* Expected output: Documents where address field contains the exact phrase "Bristol Street"
+
+Special Characters
+------------------
+The following Lucene special characters are automatically escaped in values:
+``+ - && || ! ( ) { } [ ] ^ " ~ * ? : \ /``
+
+Examples:
+
+* ``search email="user+test@example.com" source=accounts``
+  * Expected output: Documents where email contains the literal text "user+test@example.com"
+* ``search address="123 Main St. (Suite #4)" source=accounts``
+  * Expected output: Documents where address contains the literal text "123 Main St. (Suite #4)"
+
+Field Names
+-----------
+* Dots (``.``) in field names are preserved for nested fields
+* Example: ``user.name`` accesses the nested field
+
+Important Notes
+===============
+
+Wildcard Pattern Matching
+-------------------------
+Wildcards are supported for pattern matching in search expressions:
+
+* ``*`` - Matches zero or more characters
+* ``?`` - Matches exactly one character
+
+**Wildcard Examples:**
+
+* ``search severityText=ERR* source=logs`` - Matches ERROR, ERROR2, ERROR3, etc.
+* ``search name=John? source=accounts`` - Matches Johns, Johny, etc. (exactly 5 characters)
+* ``search email=*@example.com source=accounts`` - Matches any email ending with @example.com
+* ``search fail* source=logs`` - Free text search for words starting with "fail"
+
+**Escaping for Literal Matching:**
+
+To search for literal ``*``, ``?``, or ``\`` characters, you must escape them:
+
+.. list-table:: 
+   :widths: 30 35 35
+   :header-rows: 1
+
+   * - Search For
+     - PPL Syntax
+     - Example
+   * - ``prefix*`` (wildcard)
+     - ``field=prefix*``
+     - Matches "prefix", "prefixABC", etc.
+   * - ``prefix*`` (literal)
+     - ``field="prefix\\*"``
+     - Matches exactly "prefix*"
+   * - ``user?`` (wildcard)
+     - ``field=user?``
+     - Matches "user1", "userA", etc.
+   * - ``user?`` (literal)
+     - ``field="user\\?"``
+     - Matches exactly "user?"
+
+**Important Considerations:**
+* Wildcards work on both keyword and text fields
+* For text fields, wildcards apply to individual tokens after analysis
+* Leading wildcards (``*term``) may impact performance on large datasets
+
+Field Types and Search Behavior
+--------------------------------
+* **Text fields**: Analyzed for full-text search
+* **Keyword fields**: Exact value matching
+* **Numeric fields**: Support range queries (>, <, >=, <=)
+* **Date fields**: Support range queries with date values
```

Comment:
we want to call-out limitation of each data type in this section? e.g., 
* keyword field support wildcard and exactally match
* text field limitation on support wildcard search
```
{"message": "[2025-08-26] logtype=ws:access error"}

message="*=ws:access*" match nothing.
```

Reply:
@penghuo  updated the text.
Will update further for date types and any missing types.